pkg/tests/schema_pulumi_test.go

```diff
@@ -852,9 +852,7 @@ resources:
         [urn=urn:pulumi:test::test::prov:index/aux:Aux::auxRes]
     + prov:index/test:Test: (create)
         [urn=urn:pulumi:test::test::prov:index/test:Test::mainRes]
-        tests     : [
-            [0]: {}
-        ]
+        tests     : output<string>
```

Comment:
This should be an `output<[]>`, right? Is there an issue in pulumi/pulumi?

Reply:
I believe the engine only shows output<string>. I'll see if there's an issue and open one if not.

Reply:
https://github.com/pulumi/pulumi/issues/7716 is probably closest.